src/firebase-config.js

```diff
@@ -0,0 +1,22 @@
+// Import the functions you need from the SDKs you need
+import { initializeApp } from "firebase/app";
+// TODO: Add SDKs for Firebase products that you want to use
+// https://firebase.google.com/docs/web/setup#available-libraries
+import { getFirestore } from "@firebase/firestore";
+
+// Your web app's Firebase configuration
+// For Firebase JS SDK v7.20.0 and later, measurementId is optional
+const firebaseConfig = {
+  apiKey: "AIzaSyBL-6fpd-kuJMDIrJJicLDQCSfErywxJMI",
+  authDomain: "badminton-quizerria.firebaseapp.com",
+  projectId: "badminton-quizerria",
+  storageBucket: "badminton-quizerria.appspot.com",
+  messagingSenderId: "48873582359",
+  appId: "1:48873582359:web:1a1802cc60e350a29ed152",
+  measurementId: "G-QX2JRY8NMZ",
+};
```

Comment:
suggestion - you shouldn't push any kind of API keys to github, use environment variables instead.

Reply:
Sure, will make the environment variable for private data. Thanks for pointing it out.